recipes/property-cached/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "1.6.3" %}
+
+package:
+  name: property-cached
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/property-cached/property-cached-{{ version }}.zip
+  sha256: 614b6972e279d981b7bccabd0b1ce4601c1739a6eb9905fd79a9c485fb20a1e0
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - setuptools
```

Comment:
Why do you need this at runtime?

Reply:
Like I it says in the commit message, it uses `pkg_resources` :wink: